config_auth.go

```diff
@@ -121,9 +130,11 @@ func (c *AuthConfig) ValidateAndDefault() error {
 		AuthOktaKey,
 		AuthNaiveKey,
 		AuthGithubKey,
-		AuthMultiKey}, c.PreferredType), "invalid auth type '%s'", c.PreferredType)
+		AuthMultiKey,
+		AuthKanopyKey,
+	}, c.PreferredType), "invalid auth type '%s'", c.PreferredType)
 
-	if c.Naive == nil && c.Github == nil && c.Okta == nil && c.Multi == nil {
+	if c.Naive == nil && c.Github == nil && c.Okta == nil && c.Multi == nil && c.Kanopy != nil {
```

Comment:
Wait shouldn't this also be checking if it _is_ nil? Otherwise if all the auth types are nil we won't error, but if all except kanopy are nil we will error, which seems wonky

Reply:
oops 😳 
Fixed